src/main/java/ch/njol/skript/conditions/CondItemLifetime.java

```diff
@@ -0,0 +1,59 @@
+package ch.njol.skript.conditions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Condition;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import org.bukkit.entity.Entity;
+import org.bukkit.entity.Item;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Item Has Unlimited Lifetime")
+@Description("Checks if the dropped item has unlimited lifetime enabled or disabled.")
+@Examples({
+	"if all dropped items have unlimited lifetime disabled:",
+		"\tenable unlimited lifetime for all dropped items"
+})
+@Since("INSERT VERSION")
+public class CondItemLifetime extends Condition {
+
+	static {
+		Skript.registerCondition(CondItemLifetime.class, ConditionType.PROPERTY,
+			"[the] %entities% (has|have) unlimited lifetime enabled",
```

Comment:
I would like to see support for
`<entities> have unlimited lifetime`, `<entities> do not have unlimited lifetime`, etc.